Email integration package (#17849)
ckeditor/ckeditor5: packages/ckeditor5-email/tests/manual/emailintegration.md

# Email integration

> Check output in the console. Plugins that are not supported by the email integration feature should be printed in the console with a warning messages.

Reviewer: ```suggestion
Check the output in the console. It should show warnings about plugins unsupported by the email integration feature.
```
